airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/catalog_assets.py

```diff
@@ -1,15 +1,112 @@
 import pandas as pd
 import json
 from dagster import asset, OpExecutionContext
+from typing import List
 
 from ..utils.dagster_helpers import OutputDataFrame
 
 
 GROUP_NAME = "catalog"
 
+# HELPERS
```

Comment:
What do you think of keeping helpers in a separate module / package ?

Reply:
Hmm good question, and generally im totally for helper/util files/modules/classes.

We already have the helper file pattern in this orchestration project as proof 😅 

However in my mind I see helper functions as filling two possible roles
1. Providing a piece of reusable code
2. Providing a way to self document code and break up the cognitive load when reading a function that uses the helper. (code should read like a book principle) 

If a helper functions only purpose is item #2 (self documentation) but we anticipate it will only ever be used in one circumstance by one other function/class/file, then I typically think it should be collocated. in the same file.

So in this instance I would opt to leave these specific helpers here. 

Does that change your mind any?